src/components/atoms/Tab/Tab.tsx

```diff
@@ -0,0 +1,31 @@
+import {Pressable} from 'react-native';
+import {NavIcon} from '../NavIcon';
+import {BottomTabNavigationEventMap} from '@react-navigation/bottom-tabs';
+import {ParamListBase, TabNavigationState} from '@react-navigation/routers';
+import {NavigationHelpers} from '@react-navigation/core';
+import {styles} from './styles';
+
+type TTabProps = {
+  state: TabNavigationState<ParamListBase>;
+  navigation: NavigationHelpers<ParamListBase, BottomTabNavigationEventMap>;
+  route: any;
```

Comment:
to da się lepiej otypować 8)

Reply:
tego nie wiem jak otypować przynajmniej na razie a już nie chce mi się siedzieć nad tym

Reply:
dobra jednak się udało